profile/README.md

```diff
@@ -1,171 +1,18 @@
-#
+![Torrust Organization Header Image](../img/2022_06_27-Torrust_Org_Title.png)
 
-![Torrust Organization Header Image](./img/Torrust_Repo_Organization_Readme_Header-20220621.jpg)
+# Welcome to the GitHub of the __Torrust__ community-project!
 
-![Open Source](https://badgen.net/badge/Open%20Source/100%25/DA2CE7)
-![Cool](https://badgen.net/badge/Cool/100%25/FF7F50)
-![Awesome](https://badgen.net/badge/icon/awesome?icon=awesome&label)
+We develop and extend our high-quality open-source BitTorrent Related Software:
+ ### Conventioal Tracker, Written in Rust:
+ - [__torrust-tracker__ (Torrent Tracker)](https://github.com/torrust/torrust-tracker)
 
-![Language](https://badgen.net/badge/Language/Rust/green)
+### Torrent Index and Web-Interface:
 
-![Nautilus Sponsored](https://badgen.net/badge/Sponsor/Nautilus%20Cyberneering/red)
+ - [__torrust-index__ (Meta-Project)](https://github.com/torrust/torrust-index)
+  - [__torrust-index-backend__ (Core Logic and Database Management)](https://github.com/torrust/torrust-index-backend)
+ - [__torrust-index-frontend__ (Web Application)](https://github.com/torrust/torrust-index-frontend)
 
-## Index
+ 
+For more information, please consider visiting our website: __<http://torrust.org/>__.
```

Comment:
I think it is torrust.com

Reply:
I think we are moving to torrust.org for the main website and docs.torrust.org for the docs that are currently deployed on torrust.com